src/middlewares/validateBody.js

```diff
@@ -0,0 +1,14 @@
+const { httpError } = require("../helpers");
+
+const validateBody = (schema) => {
+  const func = (req, res, next) => {
+    const { error } = schema.validate(req.body);
+    if (error) {
+      next(httpError(404, error.message));
```

Comment:
```suggestion
    return  next(httpError(404, error.message));
```
В інших місцях також поправте